docs/MO_DG/prepare_model/customize_model_optimizer/Customize_Model_Optimizer.md

```diff
@@ -6,217 +6,218 @@
    :maxdepth: 1
    :hidden:
 
+   openvino_docs_MO_DG_prepare_model_customize_model_optimizer_Customize_Model_Optimizer_Model_Optimizer_Ports_Connections
+   openvino_docs_MO_DG_prepare_model_customize_model_optimizer_Model_Optimizer_Extensions
    openvino_docs_MO_DG_prepare_model_customize_model_optimizer_Extending_Model_Optimizer_With_Caffe_Python_Layers
 
```

Comment:
I think we need some section in `[Model Optimizer Usage](http://openvino-doc.iotg.sclab.intel.com/smyk-mo-extensibility-rst-test-final-2/openvino_docs_MO_DG_Deep_Learning_Model_Optimizer_DevGuide.html)` about MO extensibility or to have some link.

I don' know where to locate the legacy approach to extend. @slyalin, please advice.

Reply:
Done in https://github.com/openvinotoolkit/openvino/pull/15931/#issuecomment-1499038492